db/primary_migrate/20220411224513_add_submit_at_to_doc_auth_logs.rb

```diff
@@ -0,0 +1,10 @@
+class AddSubmitAtToDocAuthLogs < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
+
+  def change
+    add_column :doc_auth_logs, :verify_submit_at, :datetime
+    add_column :doc_auth_logs, :verify_phone_submit_count, :integer, default: 0
```

Comment:
adding a default value for a new column is expensive, right?

Reply:
Unfortunately yes but not supplying a default value broke the code a while back.  The table is actually not too large currently.  We should revisit the logic that depended at the count being populated.

Reply:
I think this is not a problem now that we're on PG13 ([docs](https://www.postgresql.org/docs/current/ddl-alter.html#DDL-ALTER-ADDING-A-COLUMN))

>From PostgreSQL 11, adding a column with a constant default value no longer means that each row of the table needs to be updated when the ALTER TABLE statement is executed. Instead, the default value will be returned the next time the row is accessed, and applied when the table is rewritten, making the ALTER TABLE very fast even on large tables.
>
>However, if the default value is volatile (e.g., clock_timestamp()) each row will need to be updated with the value calculated at the time ALTER TABLE is executed. To avoid a potentially lengthy update operation, particularly if you intend to fill the column with mostly nondefault values anyway, it may be preferable to add the column with no default, insert the correct values using UPDATE, and then add any desired default as described below.